IT.md

```diff
@@ -0,0 +1,44 @@
+# Italy
+
+## General Notes
+
+…to be filled
+
+## Postal Codes
+
+* 5 digit
+* numerical
+* no spaces
+* called "CAP"
+
+## House Number
+
+* Houses have House numbers.
```

Comment:
As far as I know, in Italy *house numbers* are assigned to entrances of houses, not houses. //cc @cortesimone.